engine-dmn/feel-juel/pom.xml

```diff
@@ -35,6 +50,13 @@
       <artifactId>operaton-juel</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <scope>test</scope>
+      <version>${version.mockito}</version>
```

Comment:
Is the version needed here? It should be actual be managed by the `internal-dependencies` BOM which ich imported by the parent. Haven't checked that, though. If required, leave it and close this thread.